### ISSUE TITLE
Notifications `oembedShown` error

### ISSUE THREAD

**Issue by sigilante** [urbit/urbit#3931]
**Describe the bug**
Notifications error.  When I load the Notifications page through any access point I see the following error:

```
Error
“Cannot read property 'oembedShown' of undefined”
Stack trace
TypeError: Cannot read property 'oembedShown' of undefined
    at r.value (http://localhost:8080/~landscape/js/bundle/index.b23d5c0044c48b1d2bab.js:184:56923)
    at Pa (http://localhost:8080/~landscape/js/bundle/index.b23d5c0044c48b1d2bab.js:37:69875)
    at Na (http://localhost:8080/~landscape/js/bundle/index.b23d5c0044c48b1d2bab.js:37:69670)
    at gs (http://localhost:8080/~landscape/js/bundle/index.b23d5c0044c48b1d2bab.js:37:105469)
    at cl (http://localhost:8080/~landscape/js/bundle/index.b23d5c0044c48b1d2bab.js:37:96716)
    at sl (http://localhost:8080/~landscape/js/bundle/index.b23d5c0044c48b1d2bab.js:37:96641)
    at Js (http://localhost:8080/~landscape/js/bundle/index.b23d5c0044c48b1d2bab.js:37:93671)
    at http://localhost:8080/~landscape/js/bundle/index.b23d5c0044c48b1d2bab.js:37:45313
    at t.unstable_runWithPriority (http://localhost:8080/~landscape/js/bundle/index.b23d5c0044c48b1d2bab.js:45:3844)
    at qi (http://localhost:8080/~landscape/js/bundle/index.b23d5c0044c48b1d2bab.js:37:45022)
```

**To Reproduce**
Unknown

**Expected behavior**
No error

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Browser: Brave
 - Base hash of your urbit ship: `tdo1o`

**Comment by matildepark** [comment]
Any guess at the type of notification? I patched this for Chat, but I wasn't aware any other notification type even embedded content. It's possible it was a link in a collection.

**Comment by sigilante** [comment]
Probably links, yeah, I've been doing a lot with those lately.

**Comment by matildepark** [comment]
This should be fixed with #3941, going out tonight
